Source/WebCore/rendering/BreakLines.h

```diff
@@ -119,16 +117,17 @@ class BreakLines {
     public:
         static constexpr UChar firstCharacter = '!';
         static constexpr UChar lastCharacter = 0xFF;
+        static constexpr unsigned rowCount = lastCharacter - firstCharacter + 1;
+        static constexpr unsigned columnCount = (lastCharacter - firstCharacter) / 8 + 1;
```

Comment:
What’s the rationale for making these public?

Reply:
In Breaklines.cpp, the `static const std::array<std::array<uint8_t, columnCount>, rowCount> breakTable;` array is defined. However, they had hard-coded the columnCount and rowCount. I wanted to re-use the same variables as in the header.